_posts/2019-10-3-ashlyn.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: Go Hacktoberfest
```

Comment:
Hi, please change your title as it is the same with another article [here](https://github.com/sunwaytechclub/hacktoberfest-2019/blob/master/_posts/2019-10-3-Go-Hacktoberfest-2019.md)